python/GafferImageUI/ImageWriterUI.py

```diff
@@ -114,69 +130,74 @@
 			"plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
 		],
 
-		"namingConvention" : [
+		"layout" : [
 
 			"description",
 			"""
-			TODO
+			Controls where channels are placed in the file, including how they are named.
+			"Single part" writes all channels to the same part ( all interleaved ).
+			"Part per layer" writes a separate part for each layer, so they may be loaded
+			independently for better performance, and is the default.  "Part per view" is
+			a compromise, where layers are not separated, but views are separated, so that
+			stereo files can use independent data windows.
+
+			There is also the option to use a layout that tries to match Nuke's default
+			conventions, including various oddities done by Nuke.  The Nuke presets
+			match "Part per Layer", "Part per View", and "Single Part", but use Nuke's
+			naming conventions.  They also do weird and incorrect Nuke things like:
+			omitting the layer name from the channel name, putting custom channels of
+			base layer in a part named "other", renaming RGB to red/green/blue in some layers.
```

Comment:
Rather than "weird" and "odd", let's go with the facts and talk about differences to the spec. How about this?

> There is also the option to use a layout that matches Nuke's default behaviour, but does not conform to the EXR specification. The Nuke presets match "Part per Layer", "Part per View", and "Single Part" respectively, but with the following deviations from the specification : 
> - The layer name is omitted from the channel name
> - Custom channels from the main layer are placed in a part named `other`
> - The RGB channels from secodary layers are renamed to `red`, `green` and `blue`